README.md

```diff
@@ -8,6 +8,142 @@ Homework 2 Repo for Software Engineering 22 Group 41
 
 Our project is the conversion of this source code: https://github.com/txt/se22/blob/main/etc/pdf/csv.pdf in lua to python.
 
+## Theory
```

Comment:
Replace this with the actual code output it is returning inline with the latest code